apps/menu-bar/electron/vite.main.config.ts

```diff
@@ -7,4 +7,12 @@ export default defineConfig({
     browserField: false,
     mainFields: ['module', 'jsnext:main', 'jsnext'],
   },
+  optimizeDeps: {
+    include: ['common-types'],
+  },
+  build: {
+    commonjsOptions: {
+      include: [/common-types/, /node_modules/],
+    },
+  },
```

Comment:
According to https://vitejs.dev/guide/dep-pre-bundling.html#monorepos-and-linked-dependencies

> In a monorepo setup, a dependency may be a linked package from the same repo. Vite automatically detects dependencies that are not resolved from node_modules and treats the linked dep as source code. It will not attempt to bundle the linked dep, and will analyze the linked dep's dependency list instead.
> However, this requires the linked dep to be exported as ESM. If not, you can add the dependency to [optimizeDeps.include](https://vitejs.dev/config/dep-optimization-options#optimizedeps-include) and [build.commonjsOptions.include](https://vitejs.dev/config/build-options#build-commonjsoptions) in your config.